app/src/main/java/org/mozilla/focus/ext/Fragment.kt

```diff
@@ -0,0 +1,11 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.ext
+
+import android.support.v4.app.Fragment
+import org.mozilla.focus.Components
+
+val Fragment.requireComponents: Components
```

Comment:
It's not shorter but I think it shows the intent better? If I see `context!!` I'm never sure whether someone just wanted to make the code compile or if they actually *assume* that this will not be null here. Naming-wise I followed `requireContext` and `requireActivity` from Fragment. And I noticed I should use `requireContext` here... :)